src/PaymentGateways/Gateways/Stripe/Helpers.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe;
+
+/**
+ * @unreleased
+ */
+class Helpers
```

Comment:
Another candidate for the `LegacyPaymentGateways` domain.

Reply:
@jonwaldstein this helper class was refactored out - no longer necessary.